QualityUnit/postaffiliatepro-theme
# [611] feat: redesign of directory page

**Changes proposed in this Pull Request**
Redesigned directory homepage.

**Testing instructions**
Go to [this](http://postaffiliatepro.local/affiliate-program-directory/) page
Check the design

**Checklist**

Close QualityUnit/web-issues#611
